arch/etc/initcpio/install/bcachefs

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+build() {
+             add_module `bcachefs`
```

Comment:
the backticks here rather than quotes causes mkinitcpio to spit out ``==> ERROR: module not found: `bcachefs:'``

switching them to quotation marks makes mkinitcpio run fine

Reply:
or no quotes at all, as is done in existing hooks. Same for `add_binary` below.